modules/signals/entities/src/entity-meta.ts

```diff
@@ -0,0 +1,23 @@
+import { SelectEntityId } from './models';
+
+export function entityMeta<Entity, Collection extends string>(meta: {
```

Comment:
**suggestion:** Rename to just `entity` instead of `entityMeta`. We don't really use the word `meta` across our API surface.

Reply:
`entity` is the property name. If we give the function the same name it would look like this:

```ts
const x = entity({ entity: type<Todo>() })
```

What do you think about `entityConfig` instead of `entityMeta`? cc @timdeschryver

```ts
const userConfig = entityConfig({
  entity: type<User>(),
  collection: 'user',
  selectId: (user) => user._id,
});
```

Reply:
I like entityConfig more than entityMeta

Reply:
I like `entityMeta` but after a second thought `entityConfig` will be clearer.

Reply:
We use `Config` generously :) I like `Config` as well.